src/shaders/globe_raster.fragment.glsl

```diff
@@ -1,9 +1,40 @@
 uniform sampler2D u_image0;
 varying vec2 v_pos0;
 
+#ifndef FOG
+uniform vec3 u_frustum_tl;
+uniform vec3 u_frustum_tr;
+uniform vec3 u_frustum_br;
+uniform vec3 u_frustum_bl;
+uniform highp vec3 u_globe_pos;
+uniform highp float u_globe_radius;
+uniform vec2 u_viewport;
+#endif
+
 void main() {
-    vec4 color = texture2D(u_image0, v_pos0);
+#if defined(CUSTOM_ANTIALIASING) && defined(USE_STANDARD_DERIVATIVES) && !defined(GLOBE_POLES)
+    vec2 uv = gl_FragCoord.xy / u_viewport;
+
+    vec3 ray_dir = mix(
+        mix(u_frustum_tl, u_frustum_tr, uv.x),
+        mix(u_frustum_bl, u_frustum_br, uv.x),
+        1.0 - uv.y);
+        
+    vec3 dir = normalize(ray_dir);
+
+    vec3 closest_point = dot(u_globe_pos, dir) * dir;
+    float norm_dist_from_center = 1.0 - length(closest_point - u_globe_pos) / u_globe_radius;
 
+    const float antialias_pixel = 4.0;
+    float antialias_factor = antialias_pixel * fwidth(norm_dist_from_center);
```

Comment:
Testing this a bit more, and with the new technique using `fwidth`, 4 pixels seem like a lot. I think we can tweak that to 2 and that would still be enough AA coverage:

<img width="918" alt="Screen Shot 2022-05-03 at 9 24 49 AM" src="https://user-images.githubusercontent.com/7061573/166495174-9419a83f-1449-4215-a0e2-b62a8395fc89.png">

```suggestion
    const float antialias_pixel = 2.0;
    float antialias_factor = antialias_pixel * fwidth(norm_dist_from_center);
```